src/server/HelloCodeaholics.Web/Program.cs

```diff
@@ -0,0 +1,36 @@
+using HelloCodeaholics.Core.Interfaces;
+using HelloCodeaholics.Infrastructure;
+using HelloCodeaholics.Services.Application;
+
+
+using Microsoft.EntityFrameworkCore;
+
+var builder = WebApplication.CreateBuilder(args);
+
+// Add services to the container.
+
+builder.Services.AddControllers();
+builder.Services.AddEndpointsApiExplorer();
+builder.Services.AddSwaggerGen();
+
+var connectionString = builder.Configuration.GetConnectionString("HelloCodeEntities");
+builder.Services.AddDbContext<HelloCodeDbContext>(options => options.UseSqlServer(connectionString));
```

Comment:
something worth checking out: [DbContextPool](https://learn.microsoft.com/en-us/ef/core/performance/advanced-performance-topics?tabs=with-di%2Cexpression-api-with-constant)

Reply:
Will check it out.  Thanks.